sdks/go/test/integration/io/fhirio/fhirio_test.go

```diff
@@ -228,6 +229,101 @@ func TestFhirIO_ExecuteBundles(t *testing.T) {
 	ptest.RunAndValidate(t, p)
 }
 
+func TestFhirIO_Search_SelectAll(t *testing.T) {
```

Comment:
Integration tests are expensive to spin up and test for each pipeline used, so I would encourage limiting the number you have as much as possible. In this situation I think you can probably consolidate all the stuff you're testing to one pipeline since they all use the same FhirStore data, just use multiple Search transforms, or consolidate testing coverage through one Search transform (ex. testing ResourceType, parameters, and pagination all with one search query). And if any of these tests work just fine as unit tests, that's a better alternative (pagination for example strikes me as suited for a unit test).